repos/fdbt-site/cypress_tests/cypress/support/helpers.ts

```diff
@@ -344,33 +344,67 @@ export const randomlyDeterminePurchaseType = (isOtherProduct?: boolean): void =>
     continueButtonClick();
 };
 
-export const randomlyDecideTimeRestrictions = (): void => {
+export const selectTimeRestriction = () => {
+    // randomly pick a time restriction
+    getElementById('conditional-time-restriction')
+        .find('[class=govuk-radios__input]')
+        .its('length')
+        .then((length) => {
+            const randomNumber = getRandomNumber(0, length - 1);
+            getElementById('conditional-time-restriction')
+                .find('[class=govuk-radios__input]')
+                .eq(randomNumber)
+                .click()
+                .then(($radio) => {
+                    const timeRestriction = $radio.attr('value');
+                    cy.wrap(timeRestriction).as('timeRestriction');
+                });
+        });
+    continueButtonClick();
```

Comment:
Should this continueButtonClick be in here? The function says selectTimeRestriction, so it might be best to have it *only* do that.